csss-site/src/events/mountain_madness/views.py

```diff
@@ -5,6 +5,11 @@
 TAB = 'events'
 
 
+def mm2021_submissions(request):
+    path = 'mountain_madness/2021/mountain_madness2021_submissions.html'
```

Comment:
why declare path rather than just pass it to `render`?

Reply:
The validator complained the line was too long so I broke it up.

Reply:
you can  just make the `render` function call multi-line instead